pkg/irmacontract.go

```diff
@@ -108,11 +141,13 @@ func (sc *SignedIrmaContract) Validate(actingPartyCn string, configuration *irma
 		return nil, err
 	}
 	if !ok {
-		verifiedContract.ValidationResult = Invalid
-		return verifiedContract, nil
+		validationResult.ValidationResult = Invalid
+		return validationResult, nil
```

Comment:
no error specified?

Reply:
Its really an error as in all went right, its only invalid.

Reply:
ok